src/utils/special-cases.ts

```diff
@@ -186,7 +186,7 @@ const specialCases: Record<string, (o: SpecialCaseOpts) => void> = {
     }
   },
   'socket.io': async function ({ id, ast, job }) {
-    if (id.endsWith('socket.io/lib/index.js')) {
```

Comment:
I think we need to keep both so we can support both the old version and the new version.